toolboxes/feel/feelmodels/modelcore/modelmeasures.cpp

```diff
@@ -275,23 +275,23 @@ ModelMeasuresStorageValues::restart( std::string const& directory, int restartIn
 {
     std::string filename = this->filename_CSV( directory );
 
-    std::unique_ptr<std::ifstream> ifile;
-    std::unique_ptr<std::ofstream> ofile;
+    std::unique_ptr<fs::ifstream> ifile;
```

Comment:
why do you need to change this ?
it is not related to the PR, is it ? 

@vincentchabannes  what do you think ?

Reply:
because it's loaded from a fs::path and @metivett has a compilation error on macos (see answer in the PR discussion).